src/Pages/Main/Cards.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Cards(props) {
+  return (
+    <div>
+      <Card>
+        <BoxImg src={props.bgImg} alt="background" />
+        <CardContainer>
+          <Title>
+            {props.title}
+            <Price>₩{props.price}원</Price>
+            <LikeText>{props.liketext}명이 좋아합니다.</LikeText>
+          </Title>
+          <ProfileBox>
+            <SubBox>
+              <Profile src={props.userImg} alt="profile" />
+              <Name>{props.name}</Name>
+              <Nick>{props.nick}</Nick>
+            </SubBox>
+            <Icon>
+              <i className="far fa-heart fa-lg"></i>
+              {/* <i class="fas fa-heart fa-lg"></i> */}
+            </Icon>
+          </ProfileBox>
+        </CardContainer>
+      </Card>
+    </div>
+  );
+}
+
+export default Cards;
+
+const Card = styled.div`
+  width: 20em;
+  height: 20em;
+  background-color: white;
+  border: 1px solid #dbdbdb;
+`;
+
+const BoxImg = styled.img`
+  width: 20em;
+  height: 200px;
+`;
+
+const Title = styled.div`
+  margin-top: 14px;
+  margin-left: 5%;
+  width: 64%;
+  height: 40px;
+  font-size: 16px;
+  color: ${props => props.theme.fontColor};
```

Comment:
```suggestion
  color: ${({theme}) => theme.fontColor};
```
매개변수자리에서 바로 구조분해 가능합니다! 나머지부분도 적용해보세요!

Reply:
수정했습니다!